source/addonHandler/__init__.py

```diff
@@ -517,26 +515,14 @@ def __init__(self, path: str):
 				_report_manifest_errors(self.manifest)
 				raise AddonError("Manifest file has errors.")
 
-	def completeInstall(self) -> Optional[str]:
-		if not os.path.exists(self.pendingInstallPath):
-			log.error(f"Pending install path {self.pendingInstallPath} does not exist")
-			return None
-

```

Comment:
please add this code back, I'm not sure why it was removed

Reply:
I've removed it when reverting #15921, since a case in which add-on is pending install and there is no directory on disk is very unlikely. Since additional logging is always nice I've added it back.